Command/RemoveCacheCommand.php

```diff
@@ -20,56 +21,89 @@
 
 class RemoveCacheCommand extends Command
 {
-    /* @var CacheManager cacheManager */
-    private $cacheManager;
+    use CacheCommandTrait;
 
-    public function __construct(CacheManager $cacheManager)
+    public function __construct(CacheManager $cacheManager, FilterManager $filterManager)
     {
-        $this->cacheManager = $cacheManager;
+        parent::__construct();
 
-        parent::__construct('liip:imagine:cache:remove');
+        $this->cacheManager = $cacheManager;
+        $this->filterManager = $filterManager;
     }
 
-    protected function configure()
+    protected function configure(): void
     {
         $this
-            ->setDescription('Remove cache for given paths and set of filters.')
-            ->addArgument('paths', InputArgument::OPTIONAL | InputArgument::IS_ARRAY, 'Image paths')
-            ->addOption(
-                'filters',
-                'f',
-                InputOption::VALUE_OPTIONAL | InputOption::VALUE_IS_ARRAY,
-                'Filters list'
-            )
+            ->setName('liip:imagine:cache:remove')
+            ->setAliases(['imagine:del'])
```

Comment:
this is a new thing, right? not sure if we should start doing this - symfony console already would find this as `l:i:c:rem` unless you have some very similar commands available.

Reply:
yes, I agree, so I removed these aliases

Reply:
thanks